### DISCUSSION TITLE
Is anyone looking into #9378 or #9410 issues ?

### DISCUSSION THREAD

**Discussion by BPC-AnonymousBeast** [encode/django-rest-framework#9454] [Potential Issue]
Referring to #9378 and #9410, I would like to know if these issues are being looked into.
If not, can anyone like mw start work on it. I mean will a PR raised related to it, be merged?

The only reason I am asking it is because in the contribution.md, it has been specifically mentioned that we need to discuss here before raising a PR or creating an Issue!!

**Comment by ishani2021** [comment]
Hi, I'd like to start working on the issue

**Reply by BPC-AnonymousBeast** [reply]
I think there is no issue anymore.
Please visit the mentioned issues once..... and let me know what you feel.

**Reply by ishani2021** [reply]
sure
